### DISCUSSION TITLE
Flynt Setup from Website to Desktop

### DISCUSSION THREAD

**Discussion by purchasingtoolpak** [flyntwp/flynt#444] [Q&A] [answered]
I am assisting a non-profit to update their existing flynt theme website. I transferred the site using wpvivid - to a fresh install of WordPress in Local WP on my desktop. The site is running fine.

The issue I am having is  "npm run start" is not working. I am unable to build the site and compile css after updating the  _base.scss and other files.

See below for error message in the Visual Studio Code terminal. Any help to get this working would be much appreciated.

'run-s' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! flynt@1.2.1 start: `run-s --silent checkVersion clean watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the flynt@1.2.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Comment by timohubois** [comment]
@purchasingtoolpak run-s is a cli command from the package [npm-run-all](https://www.npmjs.com/package/npm-run-all):
https://github.com/flyntwp/flynt/blob/70f7e75cde48241a9ff966d550226ba4782fad18/package.json#L69

Did you use Node 12 (mentioned as [dependency](https://github.com/flyntwp/flynt#dependencies)) at the installation process?

**Reply by purchasingtoolpak** [reply]
Yes, using node 12.

**Reply by timohubois** [reply]
Strange. You can try to remove the file **package-json.lock** and the folder **node_modules** and execute `npm i && npm run build`. By the way, on which operating system do you work?

**Reply by timohubois** [reply]
@purchasingtoolpak did you solve the issues?

**Reply by purchasingtoolpak** [reply]
Thanks for your response. Did not resolve issues. For some reason the package-json.lock did not regenerate. All the node modules files did generate.

**Comment by philipphaunstetter** [comment]
Just wondering - I start the local rendering with npm start - not npm run start? Just shooting out my morning thought.

**Comment by purchasingtoolpak** [comment, marked as answer]
Not sure what happened here. Decided to do a clean install of Flynt then transfer existing website. Thanks for all the responses.
